### PR TITLE
Updated gemma_2b_it CI

### PR DESCRIPTION
**Pull request by Luca-Calabria**
# What does this PR do?

This PR updated the baseline list for Gemma-2B-IT model:
1) adding "deepspeed" baseline.
2) removing the model from G1 CI because not enough memory to train it

### CODE REVIEW

**Comment by regisss** [comment]
@Luca-Calabria I guess the goal is also to perform this test right? Because we probably need to update `test_examples.py`

**Comment by jiminha** [comment]
@regisss 
With his previous PR(https://github.com/huggingface/optimum-habana/pull/1448), below tests were all added including deepspeed.  
```

<Dir optimum-habana-new>
  <Package tests>
    <Module test_examples.py>
      <UnitTestCase EagerModeCausalLanguageModelingExampleTester>
        <TestCaseFunction test_run_clm_gemma-2b-it_single_card>
      <UnitTestCase CausalLanguageModelingExampleTester>
        <TestCaseFunction test_run_clm_gemma-2b-it_single_card>
      <UnitTestCase MultiCardCausalLanguageModelingExampleTester>
        <TestCaseFunction test_run_clm_gemma-2b-it_multi_card>
      <UnitTestCase DeepspeedCausalLanguageModelingExampleTester>
        **<TestCaseFunction test_run_clm_gemma-2b-it_deepspeed>**


```
However deepspeed test is failing with 

```
>       extra_command_line_arguments = baseline.get("distribution").get(distribution).get("extra_arguments", [])
E       AttributeError: 'NoneType' object has no attribute 'get'

tests/test_examples.py:503: AttributeError****
```

**Comment by Luca-Calabria** [comment]
@regisss @libinta the PR is ready, I ran also "make style"

**Review by regisss — APPROVED** [review]

File: tests/test_examples.py [hunk 1]
```diff
@@ -326,6 +326,8 @@ def to_test(
             return True
         elif "huggyllama" in model_name and IS_GAUDI2 and deepspeed:
             return True
+        elif "gemma" in model_name and IS_GAUDI2:
```
Comment (Luca-Calabria) [inline tests/test_examples.py:329]: I updated this elif because the current flow always skip this  check

Comment (regisss) [reply tests/test_examples.py:329]: Yes you did well! But we also need to keep the former rule otherwise the deepspeed test is not registered.

Comment (Luca-Calabria) [reply tests/test_examples.py:329]: ok thank you!

**Pull request merged**